asmcomp/reloadgen.ml

```diff
@@ -113,7 +113,7 @@ method private reload i =
       let newarg = self#makeregs i.arg in
       insert_moves i.arg newarg
         (instr_cons (Iswitch(index, Array.map (self#reload) cases)) newarg [||]
-          (self#reload i.next))
+        (self#reload i.next))
```

Comment:
This change of indentation is incorrect (`(self#reload i.next)` is argument of `instr_cons`, not of `insert_moves`.